### PR TITLE
Issue 189

### LINKED ISSUE #657: [189] Issue #189 - July 15, 2021
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/main/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/main/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by jeroenleenarts**
Closes #657

### CODE REVIEW

File: _drafts/2021-07-15-issue-189.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+{% include sponsor.html %}
+
+### Starter tasks
+
+* [SR-14824](https://bugs.swift.org/browse/SR-14883) [Compiler] Consider `must` in diagnostic error message for empty `switch` cases
+
+### News and community
+
+[Bruno Roch](https://twitter.com/rockbruno_) wrote an in depth article on [How Actors Work Internally in Swift](https://swiftrocks.com/how-actors-work-internally-in-swift/).
```
Comment (fassko) [inline _drafts/2021-07-15-issue-189.md:21]:
```suggestion
[Bruno Rocha](https://twitter.com/rockbruno_) wrote an in depth article on [How Actors Work Internally in Swift](https://swiftrocks.com/how-actors-work-internally-in-swift/).
```

File: _drafts/2021-07-15-issue-189.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Tim Condon did a nice experiment with Visual Studio Code as a (server-side) Swift IDE. [He reports on his experience in a Swift Forum topic.](https://twitter.com/0xTim/status/1412775961343442954)
```
Comment (fassko) [inline _drafts/2021-07-15-issue-189.md:23]:
```suggestion
[Tim Condon](https://twitter.com/0xTim) did a nice experiment with Visual Studio Code as a (server-side) Swift IDE. [He reports on his experience in a Swift Forum topic.](https://twitter.com/0xTim/status/1412775961343442954)
```

File: _drafts/2021-07-15-issue-189.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### Accepted proposals
+
+[SE-0292](https://github.com/apple/swift-evolution/blob/main/proposals/0292-package-registry-service.md) *Package Registry Service* was [accepted](https://forums.swift.org/t/accepted-with-modifications-se-0292-package-registry-service/49849).
+
+The [3rd review](https://forums.swift.org/t/se-0292-3rd-review-package-registry-service/) of [SE-0292](https://github.com/apple/swift-evolution/blob/main/proposals/0292-package-registry-service.md) 36 has concluded. The review has been generally quiet with feedback focused on making the OpenAPI spec more robust to explicitly highlight the support for redirecting proxies which have been one of the focus points of the [2nd review](https://forums.swift.org/t/se-0292-2nd-review-package-registry-service/). As such, the proposal has been accepted with a few minor revisions.
```
Comment (fassko) [inline _drafts/2021-07-15-issue-189.md:29]:
```suggestion
The [3rd review](https://forums.swift.org/t/se-0292-3rd-review-package-registry-service/) of [SE-0292](https://github.com/apple/swift-evolution/blob/main/proposals/0292-package-registry-service.md) has concluded. The review has been generally quiet with feedback focused on making the OpenAPI spec more robust to explicitly highlight the support for redirecting proxies which have been one of the focus points of the [2nd review](https://forums.swift.org/t/se-0292-2nd-review-package-registry-service/). As such, the proposal has been accepted with a few minor revisions.
```

File: _drafts/2021-07-15-issue-189.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+[SE-0319](https://github.com/apple/swift-evolution/blob/main/proposals/0319-never-identifiable.md): *Never as Identifiable* [amendment](https://github.com/apple/swift-evolution/pull/1399) is [was under review and is accepted](https://forums.swift.org/t/se-0319-never-as-identifiable/50246).
```
Comment (fassko) [inline _drafts/2021-07-15-issue-189.md:31]:
```suggestion
[SE-0319](https://github.com/apple/swift-evolution/blob/main/proposals/0319-never-identifiable.md): *Never as Identifiable* [amendment](https://github.com/apple/swift-evolution/pull/1399) [was under review and is accepted](https://forums.swift.org/t/se-0319-never-as-identifiable/50246).
```

File: _drafts/2021-07-15-issue-189.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+### Proposals in review
+
+[SE-0314](https://github.com/apple/swift-evolution/blob/main/proposals/0314-async-stream.md): *AsyncStream and AsyncThrowingStream* is [under review](https://forums.swift.org/t/se-0314-second-review-asyncstream-and-asyncthrowingstream/49803).
+
+[SE-0304](https://github.com/apple/swift-evolution/blob/main/proposals/0319-never-identifiable.md): *Structured Concurrency* is [under fourth review](https://forums.swift.org/t/se-0304-4th-review-structured-concurrency/50281).
```
Comment (fassko) [inline _drafts/2021-07-15-issue-189.md:37]:
```suggestion
[SE-0304](https://github.com/apple/swift-evolution/blob/main/proposals/0319-never-identifiable.md): *Structured Concurrency* is [under the fourth review](https://forums.swift.org/t/se-0304-4th-review-structured-concurrency/50281).
```

**Review by fassko — CHANGES_REQUESTED** [review]
A couple of small things but overall very good!

Thank you @jeroenleenarts for helping out!

**Review by fassko — DISMISSED** [review]
Very good, thanks! 👍

**Review by fassko — APPROVED** [review]

**Review by JaredHalpern — APPROVED** [review]

**Comment by fassko** [comment]
@jeroenleenarts You can merge this, then move from `_drafts` to `_posts` and commit to master.

**Pull request merged**